packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/index.tsx

```diff
@@ -138,19 +138,28 @@ const DeviceItem = ({ device, instances, closeModalApp, backgroundRoute }: Props
         (backgroundRoute.app === 'wallet' || backgroundRoute.app === 'dashboard');
```

Comment:
As we discussed, I would called it `isDashboardOrWalletContext` as `isWalletContext` is confusing, because `isWalletContext` takes `wallet` and `dashboard` contexts into account.